frontend/src/components/accountdetails/AccountDetailsPage.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+import AccountDetailsTable from './AccountDetailsTable'
+
+export default class AccountDetailsPage extends React.Component {
+  constructor(props){
+    super(props)
+
+    this.state = {}
+
+    this.onDeleteAccount = this.onDeleteAccount.bind(this)
+    this.onDeposit = this.onDeposit.bind(this)
+    this.onWithdraw = this.onWithdraw.bind(this)
+    this.onAccountHistory = this.onAccountHistory.bind(this)
+    this.accountFilter = this.accountFilter.bind(this)
+  }
+
+  componentDidMount() {
+    this.setState({loading: true})
```

Comment:
I think that you have the same logic by using the state of the store which is much better then this one here.